src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/dnn-persona-bar-page-treeview/src/styles.less

```diff
@@ -340,4 +340,113 @@
         opacity: 0;
         height: 0px;
     }
+}
+
+
+
+/** This style is for tree scrollbar */
+body{
```

Comment:
Quite a lot of styling. Will this have to be repeated every time the scrollbar component is used?